src/threshold-ecdsa/threshold-ecdsa-introduction/introduction.md

```diff
@@ -0,0 +1,4 @@
+## Introduction
+A \\((t-n)\\) threshold signature protocol allows distributed signing among \\(n\\) participants  such that any group of \\(t+1\\) participants can produce a valid signature, while any group of fewer that \\(t\\) participants cannot. The goal is to produce signatures that are compatible with an existing centralized signature scheme. Compared to an ordinary signature scheme, the setup and signing algorithms  are replaced by interactive protocol between participants, while the signing algorithm remains identical to the verification of a signature issued by a centralized party. 
+
+With the advance of blockchain, threshold signature has received increased attention from the community. This is because transactions in blockchain are made possible via digital signatures, and it is dangerous to trust the whole signing process in a single individual, who might refuse to sign or use his signing key to benefit himself, leading to single point of failure. Hence many stakeholders are looking to perform signature generation in a distributed way. In a threshold signature scheme, an adversary cannot learn the signing key if it does not control enough number of participants, and any \\(t+1\\) participants will be able to deliver a valid signature, hence preventing the ``single point of failure'' attack above.
```

Comment:
```suggestion
With the advance of blockchain technology, threshold signature has received increasing attention from the community. This is because transactions in blockchain are made possible via digital signatures, and it is dangerous to trust the whole signing process in a single individual, who might be compromised, leading to single point of failure. Hence many stakeholders are looking to perform signature generation in a distributed way. In a threshold signature scheme, an adversary cannot learn the actual secret key if it does not control enough number of participants, and any \\(t+1\\) participants will be able to deliver a valid signature, hence preventing the ``single point of failure'' attack above.
```